Wrappers.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "FlightControlSystem.h"
+#include "SmartHomeSystem.h"
```

Comment:
Don't include what you don't use.
Use of Nested types requires including the header file where the nested type is defined.
But all other types don't.
Remove any header files that you can and add forward declarations of non-nested types. 
- [ ] revise

Reply:
I don't understand the comment. `FlightControlSystem` and `SmartHomeSystem` are used in the wrapper. So why they should be included ?